### ISSUE TITLE
Clarification request in "Configure Windows Authentication in ASP.NET Core"

### ISSUE THREAD

**Issue by Notheisz57** [dotnet/AspNetCore.Docs#27049]
Regarding the [Kestrel](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/windowsauth?view=aspnetcore-6.0&tabs=visual-studio#kestrel) section of this document, specifically the warning and info blockquotes near the start of the section.

For the warning blockquote, I was working through a new project targeting .NET 6.0 and found NTLM/Kerberos credential persistence was not enabled by default.
It may be beneficial to mention whether or not the code sample that follows results in the scenario described in the warning blockquote.
If you create a new Razor/Blazor project from template and deploy it to an IIS server, would credential persistence be an issue?
These kinds of examples are sometimes included in Microsoft Docs and I find them very informative, or at the very least they help understand the context, etc.

For the info blockquote, it is implied that if the platform/server supports Windows/Negotiate authentication natively, magic and smoke will pester you to enable the native implementation (rather than performing the authentication "in-process"?).
Could this be better detailed or an example be included to clarify what this pertains to?
If you create a new Razor/Blazor project from template and deploy it to an IIS server, would you receive this error if IIS has Windows Authentication disabled?

If clarification on these two blockquotes can be provided, I can author a change to the doc if suitable.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 401ad9a0-9e51-80a2-5846-82e9790d7257
* Version Independent ID: fbc36c26-9992-1f4c-66d3-02f898ee7ec4
* Content: [Configure Windows Authentication in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/windowsauth?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/security/authentication/windowsauth.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authentication/windowsauth.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
Is this the warning?

> Credentials can be persisted across requests on a connection. Negotiate authentication must not be used with proxies unless the proxy maintains a 1:1 connection affinity (a persistent connection) with Kestrel.

>  found NTLM/Kerberos credential persistence was not enabled by default.
It requires The [Microsoft.AspNetCore.Authentication.Negotiate](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Negotiate) NuGet package

**Comment by Notheisz57** [comment]
> Is this the warning?
> 
> > Credentials can be persisted across requests on a connection. Negotiate authentication must not be used with proxies unless the proxy maintains a 1:1 connection affinity (a persistent connection) with Kestrel.

Yes, sorry for not being clearer.  These are the info and warning block-quotes I mentioned:
![image](https://user-images.githubusercontent.com/17867971/191588684-0d7d0c6d-9e95-4df0-b4c5-da07facfc1f9.png)
